modules/tile-converter/docs/cli-reference/tile-converter.md

```diff
@@ -36,10 +48,10 @@ $ npx tile-converter --help
 | output               | \*                        | \*                        | Output folder. This folder will be created by converter if doesn't exist. It is relative to the converter path. Default: "data" folder                                                                                                               |
 | name                 | \*                        | \*                        | Tileset name. This option is used for naming in resulting json resouces and for resulting path/\*.slpk file naming                                                                                                                                   |
 | max-depth            | \*                        | \*                        | Maximal depth of the hierarchical tiles tree traversal, default: infinite                                                                                                                                                                            |
-| slpk                 | \*                        |                           | Whether the converter generate \*.slpk (Scene Layer Packages) I3S output file                                                                                                                                                                        |
+| slpk                 | \*                        |                           | Whether the converter generates \*.slpk (Scene Layer Package) I3S output file                                                                                                                                                                        |
 | 7zExe                | \*                        |                           | location of 7z.exe archiver to create slpk on Windows OS, default: "C:\\Program Files\\7-Zip\\7z.exe"                                                                                                                                                |
 | egm                  | \*                        | \*                        | location of the Earth Gravity Model (\*.pgm) file to convert heights from ellipsoidal to gravity-related format, default: "./deps/egm2008-5.pgm". A model file can be loaded from GeographicLib https://geographiclib.sourceforge.io/html/geoid.html |
-| token                | \*                        |                           | Token for Cesium ION tilesets authentication                                                                                                                                                                                                         |
+| token                | \*                        |                           | Token for I3S or Cesium ION tileset authentication.                                                                                                                                                                                                         |
```

Comment:
`token` is not supported in `I3S`->`3DTiles` conversion

Reply:
Removed I3S from here.